module-services/service-desktop/endpoints/include/endpoints/HttpEnums.hpp

```diff
@@ -30,13 +30,13 @@ namespace sdesktop::endpoints::http
 
     /*! Enum class for the HTTP methods.
      */
-    enum class Method
+    enum class Method : std::uint8_t
     {
         get = 1,
         post,
         put,
         del
```

Comment:
Consider using a capital letter for enums ;)

Reply:
Done!